app.js

```diff
@@ -192,64 +192,39 @@ app.get('/oops', function(req, res, next) {
 
 app.get('/status', (req, res, next) => res.send('CLSI is alive\n'))
 
-const resCacher = {
-  contentType(setContentType) {
-    this.setContentType = setContentType
-  },
-  send(code, body) {
-    this.code = code
-    this.body = body
-  },
-
-  // default the server to be down
-  code: 500,
-  body: {},
-  setContentType: 'application/json'
-}
-
-let shutdownTime
+let PROCESS_IS_TOO_OLD = false
```

Comment:
I'd put this on Settings.processIsTooOld. We have been putting these shutdown related flags on Settings so they can be checked in other parts of the code when needed..